app/models/manageiq/providers/kubernetes/container_manager.rb

```diff
@@ -12,7 +12,7 @@ class ManageIQ::Providers::Kubernetes::ContainerManager < ManageIQ::Providers::C
 
   include ManageIQ::Providers::Kubernetes::ContainerManagerMixin
 
-  default_value_for :port, 6443
+  DEFAULT_PORT = 6443
```

Comment:
> why do we no longer need default_value_for :port, DEFAULT_PORT?

@chessbyte You're right. The comment from the issue was:
> I think this should be a constant, EmsKubernetes::DEFAULT_PORT and have dafault_value_for use that constant. I don't see the need to instantiate an object here.

Ill change it to have both.